test/parallel/test-console-tty-colors.js

```diff
@@ -44,3 +44,27 @@ check(true, true, true);
 check(false, true, true);
 check(true, false, false);
 check(false, false, false);
+
+// check invalid colorMode type
+{
+  const stream = new Writable({
+    write: common.mustNotCall
```

Comment:
This should be `common.mustNotCall()`

Reply:
@lpinca Thanks for your comment. I fixed it.